src/pages/Main/Product/Product.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import Basket from '../../../assets/images/main/basket.png';
+import './Product.scss';
+
+const Product = props => {
+  const navigate = useNavigate();
+  const { productDetail, setIsOpenModal } = props;
```

Comment:
```suggestion
const Product = ({ productDetail, setIsOpenModal }) => {
  const navigate = useNavigate();
```

이와 같이 함수형 컴포넌트 인자자리에서 구조분해 할당이 가능합니다!